test/sequential/test-performance.js

```diff
@@ -81,34 +68,41 @@ checkNodeTiming({
   environment: { around: 0 },
   loopStart: -1,
   loopExit: -1
-});
+};
+
+function checkNodeTiming(props) {
+  Object.keys(props).forEach((prop) => {
```

Comment:
Why this change? This isn't faster or better.

Reply:
No is not, but if you are invoking a chain-able method for an array, might as well chain and iterate no?